centreon/www/class/centreonGraph.class.php

```diff
@@ -1721,12 +1761,12 @@ private function manageVMetric($vId, $vName, $indexId): void
                                 $this->rmetrics[$l_pointer]["need"] = 0;
                             }
                         }
-                    } elseif ($l_poqy->rowCount() == 0) {
+                    } elseif ($checkRealMetricStatement->rowCount() === 0) {
                         /* key : id or vname and iid */
-                        $l_poqy->closeCursor();
+                        $this->DB->closeQuery($checkRealMetricStatement);
                         $this->manageVMetric(null, $l_mnane, $lVmetric["index_id"]);
                     } else {
-                        $l_poqy->closeCursor();
+                        $this->DB->closeQuery($checkRealMetricStatement);
```

Comment:
Normal to close the connection during the loop ? You can close after, line 1772 for example.

Reply:
ok seen together